components/console/helpers/questionhelper.rst

```diff
@@ -216,6 +216,30 @@ provide a callback function to dynamically generate suggestions::
 
     The ``setAutocompleterCallback()`` method was introduced in Symfony 4.3.
 
+Do not Trim the Answer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This line must be as long as the headline, thanks

Reply:
done